css/style.css

```diff
@@ -1 +1,1461 @@
+  background-color: var(--secondary-color);
+  padding-top: 77px;
+  padding-bottom: 77px;
+  text-align: center;
+}
+
+.breadcrumbs ul {
+  margin-top: 12px;
+  display: flex;
+  justify-content: center;
+}
+
+.breadcrumbs a {
+  padding: 0 8px;
+  font-size: 14px;
+  line-height: 20px;
+}
+
+.inactive-link a {
+  color: var(--inactive-color);
+  border-right: 2px solid var(--inactive-color);
+}
+
+/********** Catalog NFT Collection ***************************/
+
+.nft-collection {
+  background-color: var(--main-background-color);
+  padding-top: 100px;
+  padding-bottom: 200px;
+}
+
+.nft-collection h2 {
+  text-align: left;
+  margin-bottom: 40px;
+}
+
+/******************** Call Action section - Coloful ***********************/
+
+.cta-section {
+  background-color: var(--secondary-color);
+}
+
+.call-action-catalog {
+  background-color: var(--secondary-color);
+  position: relative;
+  top: -100px;
+  margin: 0 255px;
+  padding: 40px 0 48px;
+  border-radius: 8px;
+  background-image: url(../img/bg-s/star-big.svg), url(../img/bg-s/star-big.svg),
+    url(../img/bg-s/star-big.svg),
+    linear-gradient(to left, rgba(35, 51, 222, 1), rgba(10, 190, 102, 1));
+  background-repeat: no-repeat;
+  background-size: 321.2px auto, 255.87px auto, 137.02px auto, cover;
+  background-position: top -60px right -60px, bottom -60px left -60px,
+    top left 350px, center;
+  background-blend-mode: screen;
+}
+
+.catalog-cta-text {
+  margin-top: 4px;
+  margin-bottom: 22px;
+  text-align: center;
+  font-family: var(--text-font);
+  color: rgba(247, 247, 249, 1);
+}
+
+.cta-catbtn-box {
+  text-align: center;
+}
+
+.cta-section .media-btn {
+  background-color: rgba(88, 101, 242, 1);
+}
+
+/********************** Catalog Footer ******************************/
+
+.catalog-footer {
+  background-color: var(--secondary-color);
+}
+
+.catalog-footer-logobox {
+  width: 324px;
+  margin-bottom: 156px;
+}
+
+.footer-cat-logo {
+  font-size: 30px;
+  line-height: 42px;
+  margin-bottom: 12px;
+}
+
+.footer-cat-logo img {
+  width: 32px;
+  height: 36px;
+}
+
+.catalog-footer-logobox p {
+  font-size: 16px;
+  line-height: 26px;
+  font-family: var(--text-font);
+}
+
+/********************** Catalog Sitemap section ***************************/
+
+.sitemap-title {
+  font-size: 16px;
+  line-height: 24px;
+  margin-bottom: 20px;
+}
+
+.sitemap-list li {
+  margin-bottom: 16px;
+}
+
+.sitemap-list a {
+  font-family: var(--text-font);
+  font-size: 14px;
+  line-height: 22px;
+  color: var(--text-color);
+}
+
+/*********************** Footer Copyright section in catalog   ******************/
+
+.footer-copyright {
+  border-top: 1px solid rgba(67, 67, 67, 1);
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+.footer-copyright h2 {
+  font-family: var(--text-font);
+  color: var(--text-color);
+  font-size: 14px;
+  line-height: 22px;
+  text-align: left;
+}
+
+.network-item-cat {
+  width: 20px;
+  height: 20px;
+  padding: 0;
+  box-sizing: content-box;
```

Comment:
I don't quite understand the reason to use `box-sizing: content-box;`. Box-sizing should be set for the entire site and cannot be overridden, with the possible exception of some specific cases.
You can read this article [https://css-tricks.com/box-sizing/](https://css-tricks.com/box-sizing/)